.env.example

```diff
@@ -0,0 +1,2 @@
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/gpdb?schema=public"
+LOG_LEVEL="debug" # Available levels: https://github.com/pinojs/pino/blob/main/docs/api.md#level-string
```

Comment:
I think I have an env variable already there

Reply:
huh?

What do you mean? 🤔

Reply:
oh I am sorry it is git ignored. I added this to .env
```
CORS_ORIGIN=http://localhost:4000
```